src/notation/view/widgets/transposedialog.cpp

```diff
@@ -52,7 +52,10 @@ TransposeDialog::TransposeDialog(QWidget* parent)
     bool rangeSelection = selection()->isRange();
     setEnableTransposeKeys(rangeSelection);
     setEnableTransposeToKey(rangeSelection);
-    setEnableTransposeChordNames(rangeSelection);
+    bool hasChordNames = std::any_of(selection()->elements().begin(), selection()->elements().end(), [](EngravingItem* item) {
+        return item->isHarmony();
+    });
+    setEnableTransposeChordNames(hasChordNames);
```

Comment:
would have been safe to simply always call with true, but this is a bit more sophisticated.  The underlying transpose logic works fine as is.